samples/cpp/requirements.txt

```diff
@@ -1,5 +1,5 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
+--extra-index-url https://storage.openvinotoolkit.org/simple/wheels/pre-release
 optimum[openvino]==1.20.0
```

Comment:
@Wovchena 
our requirements.txt looks unaligned:
- some of them mention `optimum`
- some of them mention `optimum-intel`

Reply:
Only llm_bench has `optimum-intel`